src/language/server.ts

```diff
@@ -68,7 +68,7 @@ connection.onInitialize((params: InitializeParams) => {
 connection.onInitialized(() => {
   if (hasConfigurationCapability) {
     // Register for all configuration changes.
-    connection.client.register(
+    void connection.client.register(
```

Comment:
This one probably also deserves a comment explaining that connection.client.register deals with the errors. Assuming that's the case.